setup.py

```diff
@@ -684,10 +684,15 @@ def dependencies(roots):
                 )
             )
 
-            if msc_ver >= 1913:
+            if 1913 <= msc_ver < 1929:
                 e.extra_compile_args.extend(
                     (
                         "/experimental:external",
+                    )
+                )
+            if msc_ver >= 1913:
+                e.extra_compile_args.extend(
+                    (
                         "/external:W0",
                         "/external:env:CAExcludePath",
                     )
```

Comment:
shouldn't `"/experimental:external",` be in this case too?

Reply:
No.

You only need `/experimental:external` in MSC versions between 1912 and 1929 after that it is not required.